crates/bevy_math/src/primitives/dim2.rs

```diff
@@ -3,9 +3,21 @@ use std::f32::consts::{FRAC_PI_2, FRAC_PI_3, PI};
 use super::{Measured2d, Primitive2d, WindingOrder};
 use crate::{Dir2, Vec2};
 
+#[cfg(all(feature = "serialize", feature = "bevy_reflect"))]
+use bevy_reflect::prelude::*;
+
 /// A circle primitive
 #[derive(Clone, Copy, Debug, PartialEq)]
 #[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
+#[cfg_attr(
+    feature = "bevy_reflect",
+    derive(bevy_reflect::Reflect),
+    reflect(Debug, PartialEq)
```

Comment:
Some of these types also implement `Default`, which means we should probably register `ReflectDefault` as well:

```rust
reflect(Debug, PartialEq, Default)
```